chainerrl/misc/draw_computational_graph.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import division
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import absolute_import
+from builtins import *  # NOQA
+from future import standard_library
+standard_library.install_aliases()
+
+import subprocess
+
+import chainer.computational_graph
+import chainerrl
+
+
+def collect_variables(obj):
+    """Collect Variable objects inside a given object.
+
+    Args:
+        obj (object): Object to collect Variable objects from.
+    Returns:
+        List of Variable objects.
+    """
+    variables = []
+    if isinstance(obj, chainer.Variable):
+        return [obj]
+    elif isinstance(obj, chainerrl.action_value.ActionValue):
+        return [obj.greedy_actions, obj.max]
```

Comment:
The choice of variables `[obj.greedy_actions, obj.max]` for ActionValue might not be natural to everyone.

Reply:
I agree. I chose them because 1) they are required variables of action values and 2) they are sufficient to draw a computational graph. Do you have any suggestions?

Reply:
I would like to add `params` property to `ActionValue` and to postpone drawing `ActionValue` until then.

For example, `greedy_actions` and `max` of `QuadraticActionValue` on an unbounded space are independent of the `mat` argument.

I run
```
qav = chainerrl.action_value.QuadraticActionValue(
    chainer.Variable(np.zeros((5, 5), dtype=np.float32)),
    chainer.Variable(np.ones((5, 5, 5), dtype=np.float32)),
    chainer.Variable(np.zeros((5, 1), dtype=np.float32)),
)

var = chainerrl.misc.collect_variables(qav)
chainerrl.misc.draw_computational_graph(var, 'tmpgraph')
```
- and got
  ![tmpgraph](https://user-images.githubusercontent.com/1964281/33423488-55420174-d5fc-11e7-9eec-7de6b0e22bf9.png)
- But it should have (5, 5, 5) shaped variable like
  ![tmpgraph](https://user-images.githubusercontent.com/1964281/33423459-423197ac-d5fc-11e7-8d8b-a03b62c2344e.png)

Reply:
Fair enough. I'll implement ActionValue.params.